crates/forge/tests/data/panic_decoding/Scarb.toml

```
@@ -1,7 +1,10 @@
 [package]
 name = "panic_decoding"
 version = "0.1.0"
-edition = "2023_01"
+edition = "2024_07"
 
 [dependencies]
-assert_macros = "0.1.0"
+assert_macros = "2.7.0"
```

```suggestion
assert_macros = "0.1.0"
```
`assert_macros` version aligns with `Cairo` version since `2.8.1` before it was `0.1.0`, for your reference [versions from the package registry](https://scarbs.xyz/packages/assert_macros/versions). Please update it also in other places.